From 434b41e45ef6245f9b3c87f9c4287ee1f10990c2 Mon Sep 17 00:00:00 2001
From: dopaemon <polarisdp@gmail.com>
Date: Sat, 20 Aug 2022 17:14:21 +0000
Subject: [PATCH] hybris-18.1: Make following change in rpm/dhd/helpers/util.sh

Signed-off-by: dopaemon <polarisdp@gmail.com>
---
 helpers/util.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/helpers/util.sh b/helpers/util.sh
index 1f7d619..b28d1fd 100755
--- a/helpers/util.sh
+++ b/helpers/util.sh
@@ -335,7 +335,7 @@ build() {
             -t $VENDOR-$DEVICE-$PORT_ARCH \
             $PACKAGE_TIMELINE $NO_AUTO_VERSION \
             --output-dir "$LOCAL_REPO" \
-            build >>$LOG 2>&1|| die "building of package failed"
+            build --no-check >>$LOG 2>&1|| die "building of package failed"
     done
     minfo "Building of $PKG finished successfully"
 }
--
2.37.2
